scalability_and_performance/optimization/optimizing-networking.adoc

```diff
@@ -14,14 +14,17 @@ Geneve encapsulates all tunneled traffic in user datagram protocol (UDP) packets
 inner-packets are subject to normal checksumming rules to guarantee data is not corrupted during transit. Depending on CPU performance, this additional
 processing overhead can cause a reduction in throughput and increased latency when compared to traditional, non-overlay networks.
 
-Cloud, VM, and bare metal CPU performance can be capable of handling much more than one Gbps network throughput. When using higher bandwidth links such as 10 or 40 Gbps, reduced performance can occur. This is a known issue in Geneve-based environments and is not specific to containers or {product-title}. Any network that relies on Geneve or VXLAN tunnels will perform similarly because of the tunnel implementation.
+Cloud, virtual, and bare-metal environments running {product-title} can achieve multi-gigabit network throughput. Production clusters using OVN-Kubernetes with Geneve tunneling can handle high-throughput traffic effectively without changes or special configuration. 
 
-If you are looking to push beyond one Gbps, you can:
+In some high-performance scenarios where maximum efficiency is critical, targeted performance tuning can help optimize CPU usage, reduce overhead, and ensure that you are making full use of the NIC's capabilities.
 
-* Evaluate network plugins that implement different routing techniques, such as border gateway protocol (BGP).
-* Use Geneve-offload capable network adapters. Geneve-offload moves the packet checksum calculation and associated CPU overhead off of the system CPU and onto dedicated hardware on the network adapter. This frees up CPU cycles for use by pods and applications, and allows users to utilize the full bandwidth of their network infrastructure.
+For environments where maximum throughput and CPU efficiency are critical, you can further optimize performance with the following strategies:
+
+* Validate network performance using tools such as `iPerf3` and `k8s-netperf`. These tools allow you to benchmark throughput, latency, and packets-per-second (PPS) across pod and node interfaces.
 
-Geneve-offload does not reduce latency. However, CPU utilization is reduced even in latency tests.
+* Evaluate OVN-Kubernetes User Defined Networking (UDN) routing techniques, such as border gateway protocol (BGP).
```

Comment:
Is this for the reason that with BGP, you don't need to SNAT or some other reason?